catroid/res/layout/activity_main_menu.xml

```diff
@@ -0,0 +1,89 @@
+  ~ General Public License, version 3, is available at
+  ~ http://developer.catrobat.org/license_additional_term
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  ~ GNU Affero General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Affero General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <LinearLayout
+        android:id="@+id/progress_circle"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:gravity="center"
+        android:orientation="vertical"
+        android:visibility="gone" >
+
+        <ProgressBar
+            android:id="@+id/progress_bar_header"
+            style="?android:attr/progressBarStyle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content" />
+    </LinearLayout >
+
+    <Button
+        android:id="@+id/main_menu_button_continue"
+        style="@style/MainMenuButton"
+        android:layout_height="0dp"
+        android:layout_marginTop="3dp"
+        android:layout_weight="1.5"
+        android:drawableLeft="@drawable/ic_main_menu_continue"
+        android:onClick="handleContinueButton"
+        android:text="@string/main_menu_continue" />
+
+    <Button
+        android:id="@+id/main_menu_button_new"
+        style="@style/MainMenuButton"
+        android:drawableLeft="@drawable/ic_main_menu_new"
+        android:onClick="handleNewButton"
+        android:text="@string/main_menu_new" />
+
+    <Button
+        android:id="@+id/main_menu_button_programs"
+        style="@style/MainMenuButton"
+        android:drawableLeft="@drawable/ic_main_menu_programs"
+        android:onClick="handleProgramsButton"
+        android:text="@string/main_menu_programs" />
+
+    <Button
+        android:id="@+id/main_menu_button_help"
+        style="@style/MainMenuButton"
+        android:drawableLeft="@drawable/ic_main_menu_help"
+        android:onClick="handleHelpButton"
+        android:text="@string/main_menu_help" />
+
+    <Button
+        android:id="@+id/main_menu_button_web"
+        style="@style/MainMenuButton"
+        android:drawableLeft="@drawable/ic_main_menu_community"
+        android:onClick="handleWebButton"
+        android:text="@string/main_menu_web" />
+
+    <Button
+        android:id="@+id/main_menu_button_upload"
+        style="@style/MainMenuButton"
+        android:drawableLeft="@drawable/ic_main_menu_upload"
+        android:onClick="handleUploadButton"
+        android:text="@string/main_menu_upload" />
+
+</LinearLayout>
```

Comment:
Yes, I think the only option would be to set the additional attributes programmatically (then only one layout file is needed).

Just came across that issue when I merged your code into the create@school-version, where I've added a templates button in the MainMenu --> in the worst case that would mean 4 different main_menu-layouts (2 flavors, with + without dividers each).
So, maybe the better option is that I refactor it in my branch and you just keep it as it is (as I have to do it anyway)!